Source/Glass.Mapper.Sc/SitecoreVersionAbstractions.cs

```diff
@@ -36,6 +36,7 @@ public static string GetItemUrl(Item item, UrlOptions urlOptions)
         {
 #if SC104
             if (item?.Paths == null) return string.Empty;
+            if (!item?.Paths.IsFullyQualified && item?.Parent == null) return string.Empty;

```

Comment:
You can merge the if-statements and the nullcheck on `item` is not necessary.
```
if (item?.Paths == null || (item.Paths.IsFullyQualified && item.Parent == null)) return string.Empty;
```

Reply:
Including an unit-test for this specific case might also be worth it.

Reply:
After downloading the latest I found an error on line 39 below is the corrected code
if (!item.Paths.IsFullyQualified && item?.Parent == null) return string.Empty;

Reply:
@robmaas could you get the fix checked in and we'll re-release?

Reply:
@muso31 I have fixed it in this PR. Can you please approve.
https://github.com/mikeedwards83/Glass.Mapper/pull/470/commits

Reply:
@muso31 Sorry, missed this one.
Looks like you already released it by now?

Reply:
Yep all sorted